connectors/src/connectors/microsoft/temporal/activities.ts

```diff
@@ -748,3 +760,52 @@ async function handleCsvFile(
   }
   return true;
 }
+
+async function handleTextExtraction(
```

Comment:
This is so similar to Google Drive that I'll push to make this a common function!

Reply:
Added a ticket here to refacto and share functions across GDrive and Microsoft https://github.com/dust-tt/dust/issues/6283